style.css

```
@@ -0,0 +1,228 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+main {
+  font-family: sans-serif;
+  background-image: url(./images/main_background.jpg);
+  background-position-y: -620px;
+  background-position-x: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 80vh;
+}
+
+.navbar {
+  border-bottom-left-radius: 0.5rem;
+  border-top-left-radius: 0.5rem;
+  border-bottom-right-radius: 0.5rem;
+  border-top-right-radius: 0.5rem;
```

Please refactor this line with this suggestion.
```suggestion
border-radius: 0.5rem;
```

---

Consider these changes optional.